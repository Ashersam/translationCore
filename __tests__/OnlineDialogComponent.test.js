/* eslint-env jest */
import renderer from 'react-test-renderer';
import OnlineDialog from '../src/js/components/dialogComponents/OnlineDialog';
jest.mock('material-ui/Checkbox');

// Tests for OnlineDialog React Component
describe('OnlineDialog Componenet', () => {
  test('Should render a component that matches snapshot', () => {
    const checkBoxAction = jest.fn();
    const tree = renderer.create(
        OnlineDialog(checkBoxAction)
    );
    expect(tree).toMatchSnapshot();
  });
});