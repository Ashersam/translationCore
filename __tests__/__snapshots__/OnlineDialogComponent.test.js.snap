// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnlineDialog Componenet Should render a component that matches snapshot 1`] = `
<div>
  <p
    style={
      Object {
        "fontSize": 15,
      }
    }
  >
    network_warning
  </p>
  <div
    style={
      Object {
        "display": "flex",
      }
    }
  >
    hide_warning
  </div>
</div>
`;
